apps/nextra/components/landing/sections/TestimonialsSection.tsx

```diff
@@ -13,36 +13,38 @@ interface Testimonial {
   authorLogo: ReactNode;
 }
 
+interface PlaceholderAuthorLogoProps {
+  src: string;
+}
+
+const PlaceholderAuthorLogo = ({ src }: PlaceholderAuthorLogoProps) => (
+  <Image src={src} alt="placeholder logo" height={42} width={42} />
+);
```

Comment:
Now that we have logos for each testimonial, this isn't really a placeholder element anymore. This should probably just be `AuthorLogo` and should also accept an `alt` prop so that they each have their own alt text.